cpp/src/dataframe/cudf_table.cuh

```diff
@@ -622,7 +628,9 @@ public:
   /* ----------------------------------------------------------------------------*/
   template <template <typename> class hash_function = default_hash>
   __device__ 
-  hash_value_type hash_row(size_type row_index, size_type num_columns_to_hash = 0) const
+  hash_value_type hash_row(size_type row_index,
+                           size_type num_columns_to_hash = 0,
+                           hash_value_type* initial_hash_values = nullptr) const
```

Comment:
I suggest changing the order of these arguments such that the default for `num_columns_to_hash` can be used instead of having to pass 0.

Reply:
Done